use-timescale/migration/troubleshooting.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Troubleshooting known issues
+excerpt: Troubleshooting known issues in database migrations
+products: [cloud, self_hosted]
+keywords: [backups, restore]
+tags: [recovery, logical backup, pg_dump, pg_restore]
+---
+
+# Troubleshooting known issues
+
+## Large migrations blocked
+
+Timescale's platform automatically manages the underlying disk volume. Due to
+platform limitations, it is only possible to resize the disk once every six
+hours. Depending on the rate at which you're able to copy data, you may be
+affected by this restriction. Affected instances will be unable to accept new
+data and will error with: TODO error here.
+
+If you intend on migrating more than 400&nbspGB of data to Timescale, please
+open a support case in order for the required storage to be pre-allocated in
+your database instance.
+
+## The metadata table
+
+The `_timescaledb_catalog.metadata` table contains data used in telemetry. By 
+default, this table is populated automatically. Therefore, the target database
+will most likely already contain rows. The table has a primary key constraint. 
+Dumping the data from this table out of the source database and loading it into
+the target database will cause a primary key constraint violation.
```

Comment:
@jgpruitt What are the steps to reproduce this issue?

I _am_ able to reproduce it, but only if I don't use `timescaledb_pre_restore()` in the target database.

Based on this: https://github.com/timescale/timescaledb/blob/main/sql/pre_install/tables.sql#L347-L348, I understand that _only_ the uuid will be included in the backup.

If that is truly the case, then I would suggest that we scrap this section completely. IMHO it's only a problem if you don't follow the instructions. And I would argue that the "right" fix is to use `timescaledb_pre_restore`, not to fiddle with `--exclude-table` or `--use-list`.

Reply:
yep. i agree